src/Tools.pm

```diff
@@ -401,31 +412,41 @@ DMRaid Devices look like:
 DMRaid Partitions look like:
 <strange name>_part\d
 
-
 =cut
 
-sub ReadDMRaidPartitions {
+# FIXME: this has to be done through Storage
```

Comment:
The patch doesn't change the package dependencies. The perl dep comes from the
'encoding(utf8)'.